arviz/tests/external_tests/test_data_pymc.py

```diff
@@ -405,14 +405,16 @@ class TestPyMC3WarmupHandling:
         reason="requires pymc3 3.9 or higher",
     )
     @pytest.mark.parametrize("save_warmup", [False, True])
-    def test_save_warmup(self, save_warmup):
+    @pytest.mark.parametrize("chains", [1, 2])
+    @pytest.mark.parametrize("tune,draws", [(0, 200), (100, 200), (100, 0)])
+    def test_save_warmup(self, save_warmup, chains, tune, draws):
```

Comment:
I added the `chains` parametrization because there was a problem with `chains=1` in an earlier issue.. I further reduced the number of tune/draw steps - each of the tests now runs less than 0.5 seconds.